tests/runner/kube_testplatform.go

```diff
@@ -87,6 +87,7 @@ func (c *KubeTestPlatform) addApps(apps []kube.AppDescription) error {
 		}
 		app.ImageName = fmt.Sprintf("%s:%s", app.ImageName, c.imageTag())
 
+		log.Printf("Adding app %v", app)
```

Comment:
Is just for debugging, revert file changes.

Reply:
I think it's worth keeping this.  It only adds a bit of tracing and helps the user better know how far along the test + what exactly is being deployed.  When I was first running this it was failing in deployment, and since there were no errors logs around deploy, the output pretty much just had "fail" and nothing else.  What do you think?